### PR TITLE
docs: move ingress docs to website and delete old readmes

### PR DESCRIPTION
**Pull request by aramase**
Signed-off-by: Anish Ramasekar <anish.ramasekar@gmail.com>

**Reason for Change**:

**Requirements**

- [x] squashed commits
- [x] included documentation
- [x] added unit tests and e2e tests (if applicable).

**Issue Fixed**:

**Does this change contain code from or inspired by another project?**

- [ ] Yes
- [x] No

**If "Yes," did you notify that project's maintainers and provide attribution?**

**Special Notes for Reviewers**:

### CODE REVIEW

File: website/content/en/configurations/ingress-tls.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
-# Using Secrets Store CSI and Azure Key Vault Provider to Enable NGINX Ingress Controller with TLS
+---
+type: docs
+title: "Using Secrets Store CSI and Azure Key Vault Provider to Enable NGINX Ingress Controller with TLS"
```
Comment (ritazh) [inline website/content/en/configurations/ingress-tls.md:3]: Should we shorten this title? It's really long compared to other titles

```suggestion
title: "Enable NGINX Ingress Controller with TLS"
```

File: website/content/en/configurations/ingress-tls.md [hunk 2]
```diff
@@ -32,13 +38,20 @@ az keyvault certificate import --vault-name $AKV_NAME -n $CERT_NAME -f $CERT_NAM
...
+helm repo add csi-secrets-store-provider-azure https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/charts
+helm install csi csi-secrets-store-provider-azure/csi-secrets-store-provider-azure
+```
+
+Refer to [installation](../../getting-started/installation) for more details and validation.
 
 ### Optional: Deploy AAD Pod Identity
 
-If using AAD pod identity to access Azure Key Vault, make sure it is [configured properly](https://azure.github.io/aad-pod-identity/docs/demo/standard_walkthrough/) in the cluster.
+If using AAD pod identity to access Azure Keyvault, make sure it is [configured properly](https://azure.github.io/aad-pod-identity/docs/demo/standard_walkthrough/) in the cluster. Refer to [doc](identity-access-modes/pod-identity-mode) on how to use AAD Pod identity to access keyvault.
```
Comment (ritazh) [inline website/content/en/configurations/ingress-tls.md:54]: bad link: identity-access-modes/pod-identity-mode

File: website/content/en/configurations/ingress-tls.md [hunk 3]
```diff
@@ -55,23 +68,25 @@ kubectl create ns $NAMESPACE
 
 ### Create the SecretProviderClass
 
-* To provide identity to access key vault, refer to the following [section](https://github.com/Azure/secrets-store-csi-driver-provider-azure#provide-identity-to-access-key-vault).
+* To provide identity to access key vault, refer to the following [section](identity-access-modes).
```
Comment (ritazh) [inline website/content/en/configurations/ingress-tls.md:71]: bad link

File: website/content/en/configurations/ingress-tls.md (hunk 1 above)
Comment (aramase) [reply website/content/en/configurations/ingress-tls.md:3]: yeah, sounds good!

File: website/content/en/configurations/ingress-tls.md [hunk 4]
```diff
@@ -55,23 +68,25 @@ kubectl create ns $NAMESPACE
...
 * Set the `tenantId` and `keyvaultName`
 * If using **AAD pod identity** to access Azure Key Vault - set `usePodIdentity: "true"`
-* Use `objectType: secret` for the certificate, as this is the only way to retrieve the certificate and private key from azure key vault as documented [here](https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/docs/getting-certs-and-keys.md#getting-certificates-and-keys-using-azure-key-vault-provider)
+* Use `objectType: secret` for the certificate, as this is the only way to retrieve the certificate and private key from azure key vault as documented [here](getting-certs-and-keys)
```
Comment (ritazh) [inline website/content/en/configurations/ingress-tls.md:74]: bad link

File: website/content/en/configurations/ingress-tls.md [hunk 5]
```diff
@@ -147,7 +162,7 @@ controller:
 EOF
 ```
 
-If not using [service principal mode](./service-principal-mode.md), remove the following snippet from the script:
+If not using [service principal mode](./identity-access-modes/service-principal-mode), remove the following snippet from the script:
```
Comment (ritazh) [inline website/content/en/configurations/ingress-tls.md:165]: link

File: website/content/en/configurations/ingress-tls.md [hunk 6]
```diff
@@ -185,7 +200,7 @@ Depending on the TLS certificate lifecycle, follow one of the following steps:
               name: secrets-store-creds
 ```
 
-If not using [service principal mode](./service-principal-mode.md), remove the following snippet from [deployment-app-one.yaml](./sample/ingress-controller-tls/deployment-app-one.yaml) and [deployment-app-two.yaml](./sample/ingress-controller-tls/deployment-app-two.yaml)
+If not using [service principal mode](./identity-access-modes/service-principal-mode), remove the following snippet from [deployment-app-one.yaml](https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/docs/sample/ingress-controller-tls/deployment-app-one.yaml) and [deployment-app-two.yaml](https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/docs/sample/ingress-controller-tls/deployment-app-two.yaml)
```
Comment (ritazh) [inline website/content/en/configurations/ingress-tls.md:203]: link

Comment (aramase) [reply website/content/en/configurations/ingress-tls.md]: thank you! All the links should be fixed now.

**Review by ritazh — APPROVED** [review]

**Pull request merged**